internal/provider/resource_group_members_test.go

```diff
@@ -0,0 +1,247 @@
+package googleworkspace
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccResourceGroupMembers_basic(t *testing.T) {
+	t.Parallel()
+
+	domainName := os.Getenv("GOOGLEWORKSPACE_DOMAIN")
+
+	if domainName == "" {
+		t.Skip("GOOGLEWORKSPACE_DOMAIN needs to be set to run this test")
+	}
+
+	testGroupVals := map[string]interface{}{
+		"domainName": domainName,
+		"userEmail":  fmt.Sprintf("tf-test-%s", acctest.RandString(10)),
+		"groupEmail": fmt.Sprintf("tf-test-%s", acctest.RandString(10)),
+		"password":   acctest.RandString(10),
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: providerFactories,
+		CheckDestroy: resource.ComposeTestCheckFunc(
+			testAccResourceGroupMembersExists("googleworkspace_group_members.my-group-members"),
+		),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccResourceGroupMembers_basic(testGroupVals),
+			},
+			{
+				ResourceName:      "googleworkspace_group_members.my-group-members",
+				ImportState:       true,
+				ImportStateVerify: true,
```

Comment:
I think we want to add in an `ImportStateVerifyIgnore` here for `members.0.delivery_settings` (and `members.1.delivery_settings` too in the `_full` test), since these won't be returned on import, and thus won't match the prior state.

Reply:
Done